.travis.yml

```diff
@@ -10,6 +10,8 @@ sudo: required
 
 language: python
 
+python: "2.7"
```

Comment:
Why don't we specify all the versions we support? I believe we support 2.7, 3.5, and 3.6.

Reply:
This will actually create many builds. Just 2.7 is fine for now.

Reply:
Simon is right. Currently the Travis is used only for the Kubernetes integration tests.